verge3.html

```diff
@@ -0,0 +1,437 @@
+        <section class="row1 row">
+            <header>
+                <h2 class="row_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+            <p> By Gordan Jolsan
+                <span class="p-comment-count heat2">     14
+                              
+                            </span>
+            </p>
+        </section>
+        <section class="row2 row">
+            <header>
+                <h2 class="row_h2"> Twitter now lets any user filter out threats and abuse</h2></header>
+            <p>By Rick Cormick.
+                <span class="p-comment-count heat2">     14
+                              
+                            </span>
+        </section>
+        <section class="row3 row">
+            <header>
+                <h2 class="row_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+            <p>By Tasha Robinson
+                <span class="p-comment-count heat2">     14
+                              
+                            </span>
+            </p>
+        </section>
+        <section class="image_scroll">
+            <div class="image_scroll_list" id="scroll_image1">
+                <header>
+                    <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+            <div class="image_scroll_list" id="scroll_image2">
+                <header>
+                    <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+            <div class="image_scroll_list" id="scroll_image1">
+                <header>
+                    <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2></header>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+            <div class="image_scroll_list" id="scroll_image2">
+                <h2 class="image_scroll_list_h2"> Twitter now lets any user filter out threats and abuse </h2>
+                <p>By Tasha Robinson
+                    <span class="p-comment-count heat2">     14
+                              
+                                                    </span>
+                </p>
+            </div>
+        </section>
+        <div class="wrapper_video">
+            <div class="video_header">
```

Comment:
Try not to use too many nested div.(102-113)
Avoid div in video_header_image, caption1 and caption2.
Use a, h, span tags instead.